server/resource.go

```diff
@@ -142,19 +142,24 @@ func (s *Server) registerResourceRoutes(g *echo.Group) {
 			if err != nil && common.ErrorCode(err) != common.NotFound {
 				return echo.NewHTTPError(http.StatusInternalServerError, "Failed to find local storage path setting").SetInternal(err)
 			}
-			localStoragePath := "assets/{timestamp}_{filename}"
+			localStoragePath := "assets/{publicid}"
 			if systemSettingLocalStoragePath != nil {
-				err = json.Unmarshal([]byte(systemSettingLocalStoragePath.Value), &localStoragePath)
+				var s string
+				err = json.Unmarshal([]byte(systemSettingLocalStoragePath.Value), &s)
```

Comment:
Why not just use `localStoragePath` directly?

Reply:
There's a bug if use `localStoragePath` directly.

If I set a valid `localStoragePath` in System Settings, and then reset it to empty. The expect value of `localStoragePath` should be the default value of `assets/{publicid}`. But if you directly unmarshal to `localStoragePath`, it will be a value of empty string, instead of default `assets/{publicid}`